.buildkite/pipeline.yml

```diff
@@ -281,7 +281,7 @@ steps:
     timeout_in_minutes: 20
     depends_on:
       - "appium-test-fixture"
-      - "push-branch-cli-legacy"
+      - "push-branch-cli"
```

Comment:
Corrects a mistake from an earlier PR - the build for that only passed by chance because the step it needs happened to have finished.

Reply:
d'oh!